schedule/qam/common/qem-live-migration.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: live_migration
+description:    >
+    scheduling tests of live migration
+
+schedule:
+   - autoyast/prepare_profile
+   - boot/boot_from_pxe
+   - autoyast/installation
+   - virt_autotest/login_console
+   - '{{migrate_src}}'
+   - '{{migrate_dst}}'
+conditional_schedule:
+   migrate_src:
+      MIGRATION_ROLE:
+         src:
+            - virt_autotest/qem/live_migration_src
+   migrate_dst:
+      MIGRATION_ROLE:
+         dst:
+            - virt_autotest/qem/live_migration_dst
+
+test_data:
+   location:
+      "sles-15-sp1": "http://dist.suse.de/install/SLP/SLE-15-SP1-Installer-LATEST/x86_64/DVD1/"
+      "sles-15-sp2": "http://dist.suse.de/install/SLP/SLE-15-SP2-Full-GM/x86_64/DVD1/"
+      "sles-15-sp3": "http://dist.suse.de/install/SLP/SLE-15-SP3-Full-GM/x86_64/DVD1/"
+      "sles-12-sp5": "http://dist.suse.de/install/SLP/SLE-12-SP5-Server-LATEST/x86_64/DVD1/"
```

Comment:
We already have `%virt_autotest::common::guests`.

Reply:
common::guests pre-defines many data but they are generated automatically in this test such as ip, mac and guest aotuyast profile. 
Suppose a new version comes in, you have to create a profile manually. This test will generate it automatically from template.

Reply:
common::guests pre-defines many data but they are generated automatically in this test such as ip, mac and guest aotuyast profile. 
Suppose a new version comes in, you have to create a profile manually. This test will generate it automatically from template.

Reply:
This is too big change - please reuse what is already done.
First of all I don't see any point of generating the guests address.
Second of all that is not the purpose of this pull request.

Reply:
After guest live migration, we need verify that guest is working, behaving all the same way as it was before migration
i.e. the ip address is still the same? It's reachable? the services in guest are still running? ...
Users should not feel any change. That's what we test it for.  
Automation must have guest IP so it can validate.

Reply:
Yes, it has the guest's IP in `%virt_autotest::common::guests` and we also have `virt_autotest::utils::is_guest_online()`.